tasks/eth/021-holocene-protocol-versions/SignFromJson.s.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson as OriginalSignFromJson} from "script/SignFromJson.s.sol";
+import {ProtocolVersionsBump} from "script/verification/ProtocolVersionsBump.s.sol";
+import {SuperchainRegistry} from "script/verification/Verification.s.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {Proxy} from "@eth-optimism-bedrock/src/universal/Proxy.sol";
+import {SystemConfig} from "@eth-optimism-bedrock/src/L1/SystemConfig.sol";
+import {ProtocolVersions, ProtocolVersion} from "@eth-optimism-bedrock/src/L1/ProtocolVersions.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import {LibString} from "solady/utils/LibString.sol";
+
+contract SignFromJson is OriginalSignFromJson, ProtocolVersionsBump {
+    constructor()
+        SuperchainRegistry("mainnet", "op", "v1.8.0-rc.4")
+        ProtocolVersionsBump(vm.envAddress("OWNER_SAFE"), ProtoVer(9, 0, 0, 0), ProtoVer(9, 0, 0, 0))
+    {}
+
+    /// @notice Checks the correctness of the deployment
+    function _postCheck(Vm.AccountAccess[] memory accesses, Simulation.Payload memory /* simPayload */ )
+        internal
+        view
+        override
+    {
+        console.log("Running assertions");
+        checkStateDiff(accesses);
+        checkProtocolVersions();
+        console.log("All assertions passed!");
+    }
+
+    function getAllowedStorageAccess() internal view override returns (address[] memory) {
+        return allowedStorageAccess;
+    }
+
+    function getCodeExceptions() internal view override returns (address[] memory) {
```

Comment:
Seems the `getCodeExceptions()` is not called by anywhere is that intended?

Reply:
Good spot. I removed it.